src/main/java/seedu/duke/parser/Parser.java

```diff
@@ -83,6 +90,9 @@ static Command parse(String input) throws WrongFormatException,
             }
             throw new WrongFormatException();
         case "purge":
+            if (inputWords.length > 1){
```

Comment:
Perhaps put a space in between the closing parenthesis and the opening curly braces